src/entity/ClaimableItem.ts

```diff
@@ -35,6 +35,6 @@ export class ClaimableItem {
   @Column({ nullable: true })
   claimedById?: string;
 
-  @ManyToOne('User', { lazy: true })
+  @ManyToOne('User', { lazy: true, onDelete: 'SET NULL' })
```

Comment:
Should we delete the entity here too, when the user deletes their account?

Reply:
yes we should